pkgs/applications/version-management/sourcehut/dispatch.nix

```diff
@@ -21,10 +21,9 @@ buildPythonPackage rec {
 
   preBuild = ''
     export PKGVER=${version}
+    export SRHT_PATH=${srht}/${python.sitePackages}/srht
```

Comment:
Looks like you need to bring in python at the top for this.

```
    24|     export SRHT_PATH=${srht}/${python.sitePackages}/srht
      |                                ^
    25|   '';

undefined variable 'python'
```

Reply:
Thanks!